content/docs/release-notes/2017-11-01.md

```diff
@@ -7,4 +7,4 @@ releaseType:
 
 ###	Marketing Campaigns Contacts
 
-The Contacts page in Marketing Campaigns has been updated to improve the user experience! The UI has been simplified to make adding contacts more streamlined and easy-to-use. For more information, see the[adding contacts](/ui/managing-contacts/adding-contacts/).
+The Contacts page in Marketing Campaigns has been updated to improve the user experience! The UI has been simplified to make adding contacts more streamlined and easy-to-use. For more information, see the [adding contacts](/ui/managing-contacts/adding-contacts/).
```

Comment:
Can you please delete "the" before the link to make the sentence grammatically correct? Thanks!